docs/workforce-management-framework/topics/pro-overview-introduction.adoc

```diff
@@ -0,0 +1,99 @@
+== Introduction
+
+{WFM-RC-NameLong}  consist of set of node.js modules written in TypeScript language that are packaged and distributed as
```

Comment:
That statement is not fully true. replace `node.js` with `npm modules` - Node.js suggest that our modules are server side while majority of them are running in the browser.